test-data/unit/fixtures/f_string.pyi

```diff
@@ -0,0 +1,41 @@
+# Builtins stub used for format-string-related test cases.
+# We need str and list, and str needs join and format methods.
+
+from typing import TypeVar, Generic, builtinclass, Iterable, Iterator, List, overload
+
+T = TypeVar('T')
+
+@builtinclass
```

Comment:
Curious, what does this do? I haven't seen it before and grep shows no uses outside of test fixtures.